weapon/CWeaponInfo.hpp

```diff
@@ -209,15 +210,60 @@ class CWeaponInfo : public CItemInfo
 	float m_network_headshot_modifier; //0x0284
 	float m_lock_on_range; //0x0288
 	float m_weapon_range; //0x028C
-	char pad_0290[8]; //0x0290
+	float m_ai_sound_range; //0x0290
+	float m_ai_potential_blast_event_range; //0x0290
 	float m_damage_fall_off_range_min; //0x0298
 	float m_damage_fall_off_range_max; //0x029C
 	float m_damage_fall_off_modifier; //0x02A0
-	char pad_02A4[64]; //0x02A4
+	uint32_t m_vehicle_weapon_hash; //0x02B4
```

Comment:
Jump from 0x2A0 to 0x2B4 is too large.

0x2B4 should be 0x2A4.

The final size should be 0x5E8.